### PR TITLE
update documentation on deprecated functionality in (generic) easyblocks

### PR DESCRIPTION
**Pull request by boegel**
rendered version at https://boegel-eb.readthedocs.org/en/depr_easyblocks/Deprecated-functionality.html#mrbayes-beagle and https://boegel-eb.readthedocs.org/en/depr_easyblocks/Deprecated-functionality.html#changes-generic-easyblocks

@wpoely86: please review (cfr. https://github.com/hpcugent/easybuild-easyblocks/pull/547)

### CODE REVIEW

File: docs/Deprecated-functionality.rst [hunk 1]
```diff
@@ -240,6 +242,23 @@ be a known license type (see ``eb --avail-easyconfig-licenses``).
...
+* *deprecated since:* EasyBuild v1.6.0 (Jul'14)
+* *no longer supported in:* EasyBuild v2.0
+* *alternative(s)*: use ``beagle-lib`` instead
+
+Due to a misnomer in the easyconfig files for ``beagle-lib`` (formerly named ``BEAGLE``), the custom easyblock for
+``MrBayes`` now no longer considers ``BEAGLE`` as a dependency.
+
+The library required by ``MrBayes`` must now be provided as a dependency named ``beagle-lib``.
+
+
```
Comment (fgeorgatos) [inline docs/Deprecated-functionality.rst]: "I knew that HPCBIOS_Bioinfo was a mess, but not how much of a mess it was" - underneath it ; no-op for you here ;-)

File: docs/Deprecated-functionality.rst [hunk 2]
```diff
@@ -403,3 +422,50 @@ Some functions have moved to a different location:
...
+* *deprecated since:* EasyBuild v1.4.0 (May'13)
+* *no longer supported in:* EasyBuild v2.0
+* *alternative(s)*: equivalent ``srcdir`` named argument
+
+Since the ``builddir`` named argument in the ``configure_step`` method of the generic ``CMakeMake`` easyblock was a
+misnomer (it specifies the location of the *source* directory that should be provided to ``cmake``), it was replaced
+with an equivalent named argument ``srcdir``.
+
+``VersionIndependentPythonPackage`` replaces ``VersionIndependentPythonPackage``
```
Comment (fgeorgatos) [inline docs/Deprecated-functionality.rst]: eh? where is the `d` vs `t`point?

**Comment by wpoely86** [comment]
looks fine, no remarks.

File: docs/Deprecated-functionality.rst [hunk 3]
```diff
@@ -403,3 +422,50 @@ Some functions have moved to a different location:
...
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+**The** ``VersionIndependendPythonPackage`` **generic easyblock was replaced with the equivalent**
+``VersionIndependentPythonPackage`` **easyblock.**
```
Comment (fgeorgatos) [inline docs/Deprecated-functionality.rst]: we miss an adjective here? ("non-generic" perhaps?)

**Comment by fgeorgatos** [comment]
reviewed; overall pretty ok.

File: docs/Deprecated-functionality.rst (hunk 2 above)
Comment (boegel) [inline docs/Deprecated-functionality.rst]: ah, typo in the name with a typo :) will fix, thx

File: docs/Deprecated-functionality.rst (hunk 3 above)
Comment (boegel) [inline docs/Deprecated-functionality.rst]: both are generic, will fix

File: docs/Deprecated-functionality.rst (hunk 1 above)
Comment (boegel) [inline docs/Deprecated-functionality.rst]: :)

**Comment by boegel** [comment]
Thanks for the review @fgeorgatos and @wpoely86!

**Pull request merged**